dev/releases/make_archives.py

```diff
@@ -130,37 +141,40 @@
     # now create the file package-infos.json
     # We first build the json package, then create the package-infos.json, then
     # clean up the json package again.
+    notice("Compiling json package")
     path_to_json_package = glob.glob(f'{tmpdir}/{basename}/pkg/json*')[0]
     with working_directory(path_to_json_package):
         subprocess.run(["./configure"], check=True)
         subprocess.run(["make"], check=True)
 
+    notice("Constructing package_infos json file")
     package_infos = subprocess.run(["./bin/gap.sh", "-r", "--quiet", "--quitonbreak",
                                     "dev/releases/PackageInfos-to-JSON.g"],
                                     check=True, capture_output=True, text=True)
-    package_infos = package_infos.stdout
-
     with working_directory(tmpdir):
         with gzip.open("package-infos.json.gz", 'wb') as file:
-            file.write(package_infos.encode('utf-8'))
+            file.write(package_infos.stdout.encode('utf-8'))
+        manifest_list.append("package-infos.json.gz")
 
+    notice("Cleaning up the json package")
     with working_directory(path_to_json_package):
         subprocess.run(["make", "clean"], check=True)
-        subprocess.run(["rm", "-rf", "bin/", "Makefile"],
-                       check=True)
+        subprocess.run(["rm", "-rf", "bin/", "Makefile"], check=True)
 
 
-    notice("Building the manuals")
+    notice("Building GAP's manuals")
     run_with_log(["make", "doc"], "gapdoc", "building the manuals")
 
     notice("Removing unwanted version-controlled files")
     badfiles = [
     ".appveyor.yml",
+    ".clang-format",
     ".codecov.yml",
+    ".ctags",
+    ".editorconfig",
```

Comment:
Is it correct to remove these files from a GAP release? I was unsure, please let me know if I should remove anything from this list.

Reply:
Note that I intentionally left in .clang-format, .editoconfig and .ctags precisely because someone might edit the code in there and submit the result as a patch to us -- in that case, I'd prefer if they used settings similar to what we ask for.

That said, of course it's not clear whether people who don't use our git repo to make a PR would then go on to use clang-format (plus, these days that's not going to be many people, if any at all).

So, I guess then removing these is OK *shrug*

Reply:
Sure. They're hidden files anyway by default so they're barely 'clutter', and possibly useful. I'll remove the diff the next time I push.